client/landing/stepper/declarative-flow/internals/steps-repository/domains/index.tsx

```diff
@@ -291,186 +204,30 @@ const DomainsStep: Step = function DomainsStep( { navigation, flow } ) {
-				initialState.hideInitialQuery = ! domainSearchInQuery;
-			}
-		}
-
-		if ( 'undefined' === typeof showExampleSuggestions ) {
-			showExampleSuggestions = true;
-		}
-
-		return (
-			<CalypsoShoppingCartProvider>
-				<RegisterDomainStep
-					analyticsSection={ getAnalyticsSection() }
-					basePath={ path }
-					deemphasiseTlds={ flow === 'ecommerce' ? [ 'blog' ] : [] }
-					designType={ undefined }
-					domainsWithPlansOnly={ domainsWithPlansOnly }
-					includeDotBlogSubdomain={ shouldIncludeDotBlogSubdomain() }
-					includeWordPressDotCom={ includeWordPressDotCom ?? true }
-					initialState={ initialState }
-					isPlanSelectionAvailableInFlow={ isPlanSelectionAvailableLaterInFlow }
-					isReskinned={ true }
-					reskinSideContent={ getSideContent() }
-					isSignupStep
-					key="domainForm"
-					mapDomainUrl={ getUseYourDomainUrl() }
-					offerUnavailableOption
-					otherManagedSubdomains={ getOtherManagedSubdomains() }
-					otherManagedSubdomainsCountOverride={ getOtherManagedSubdomainsCountOverride() }
-					onAddDomain={ handleAddDomain }
-					onAddMapping={ handleAddMapping }
-					onSave={ setDomainForm }
-					onSkip={ handleSkip }
-					path={ path }
-					products={ productsList }
-					promoTlds={ getPromoTlds() }
-					selectedSite={ selectedSite }
-					showExampleSuggestions={ showExampleSuggestions }
-					showSkipButton={ showSkipButton }
-					suggestion={ initialQuery }
-					transferDomainUrl={ getUseYourDomainUrl() }
-					handleClickUseYourDomain={ () => setShowUseYourDomain( true ) }
-					vendor={ getSuggestionsVendor( {
-						isSignup: true,
-						isDomainOnly: false,
-						flowName: flow || undefined,
-					} ) }
-				/>
-			</CalypsoShoppingCartProvider>
-		);
-	};
-
-	const renderContent = () => {
-		let content;
-
-		if ( showUseYourDomain ) {
-			content = renderYourDomainForm();
-		} else {
-			content = renderDomainForm();
-		}
-
-		// if ( 'invalid' === this.props.step.status ) {
-		// 	content = (
-		// 		<div className="domains__step-section-wrapper">
-		// 			<Notice status="is-error" showDismiss={ false }>
-		// 				{ this.props.step.errors.message }
-		// 			</Notice>
-		// 			{ content }
-		// 		</div>
-		// 	);
-		// }
-
-		return (
-			<div className="domains__step-content domains__step-content-domain-step">{ content }</div>
-		);
-	};
+	const renderContent = () => (
+		<DomainFormControl
+			analyticsSection={ getAnalyticsSection() }
+			flow={ flow }
+			onAddDomain={ handleAddDomain }
+			onAddMapping={ handleAddMapping }
+			onAddTransfer={ handleAddTransfer }
+			onSkip={ handleSkip }
+			onUseYourDomainClick={ () => setShowUseYourDomain( true ) }
+			showUseYourDomain={ showUseYourDomain }
+		/>
+	);
 
 	return (
 		<StepContainer
 			stepName="domains"
 			isWideLayout={ true }
-			hideBack={ true }
+			shouldStickyNavButtons={ isCopySiteFlow( flow ) }
+			hideBack={ ! isCopySiteFlow( flow ) }
```

Comment:
We can considere changing the text to "Back to Sites".

```suggestion
			hideBack={ ! isCopySiteFlow( flow ) }
			backLabelText={ __( 'Back to Sites' ) }
```
<img width="1382" alt="Screenshot 2023-02-06 at 13 20 08" src="https://user-images.githubusercontent.com/779993/216982461-f1895777-793f-49e3-b195-a7f59f5d0403.png">